docs/static/command-line-flags.asciidoc

```diff
@@ -41,42 +41,47 @@ Logstash has the following flags. You can use the `--help` flag to display this
 --debug
  Increase verbosity to the last level (trace), more verbose.
 
---debug-config
+--debug.config
  Print the compiled config ruby code out as a debug log (you must also have --debug enabled).
  WARNING: This will include any 'password' options passed to plugin configs as plaintext, and may result
  in plaintext passwords appearing in your logs!
 
 -V, --version
   Display the version of Logstash.
 
--p, --pluginpath
+-p, --plugin.path
   A path of where to find plugins. This flag can be given multiple times to include
   multiple paths. Plugins are expected to be in a specific directory hierarchy:
   'PATH/logstash/TYPE/NAME.rb' where TYPE is 'inputs' 'filters', 'outputs' or 'codecs'
   and NAME is the name of the plugin.
 
--t, --configtest
+-t, --config.test
   Checks configuration and then exit. Note that grok patterns are not checked for
   correctness with this flag.
   Logstash can read multiple config files from a directory. If you combine this
   flag with `--debug`, Logstash will log the combined config file, annotating the
   individual config blocks with the source file it came from.
   
--r, --[no-]auto-reload
+-r, --auto.reload
   Monitor configuration changes and reload the configuration whenever it is changed.
 
---allow-env
+--allow.env
   EXPERIMENTAL: Enable environment variable templating within configuration parameters.
   
---reload-interval RELOAD_INTERVAL
+--reload.interval RELOAD_INTERVAL
```

Comment:
Should this (and others) match the same setting name? By this, I mean, should this flag be `--config.reload.interval` ?